modules/caddyhttp/caddyauth/command.go

```diff
@@ -52,16 +60,43 @@ be provided (scrypt).
 }
 
 func cmdHashPassword(fs caddycmd.Flags) (int, error) {
+	var err error
+
 	algorithm := fs.String("algorithm")
 	plaintext := []byte(fs.String("plaintext"))
 	salt := []byte(fs.String("salt"))
 
 	if len(plaintext) == 0 {
-		return caddy.ExitCodeFailedStartup, fmt.Errorf("password is required")
+		if terminal.IsTerminal(int(os.Stdin.Fd())) {
+			var confirmation []byte
+
+			fmt.Print("Plaintext: ")
+			plaintext, err = terminal.ReadPassword(int(os.Stdin.Fd()))
+			fmt.Println()
+
+			fmt.Print("Confirm Plaintext: ")
+			confirmation, err = terminal.ReadPassword(int(os.Stdin.Fd()))
+			fmt.Println()
+
+			if !bytes.Equal(plaintext, confirmation) {
+				return caddy.ExitCodeFailedStartup, fmt.Errorf("plaintext does not match")
+			}
```

Comment:
What's a good, idiomatic way to handle these errors?

Reply:
I think just
```go
if err != nil {
	return caddy.ExitCodeFailedStartup, err
}
```

Reply:
There's no error for this though; it needs to be made.

How about `fmt.Errorf("passwords do not match")`

Reply:
> There's no error for this though; it needs to be made.

Specifically, in the `ReadPassword` call. I guess I should just move/copy the error block from the bottom.

> How about `fmt.Errorf("passwords do not match")`

I was leaning towards `plaintext` for consistency with the CLI arg.